### PR TITLE
Added the ability to save the device registration to rock.

### PR DESCRIPTION
**Pull request by richarddubay**
# Feature / Fixed Issue(s)
- Added the ability to save the device registration to Rock.

### CODE REVIEW

**Comment by jbaxleyiii** [comment]
**[Storybook Hub](https://hub.getstorybook.io/)** is building storybooks for every commit in this PR.

**Latest Storybook: https://bac74f49-9408-4c21-9886-0432322661ff.sbook.io/**
<sub>( Visit [here](https://hub.getstorybook.io/app/github.com/NewSpring/Holtzman/branch/save-device-registration/) for more info )</sub>

> _This comment is automatically added by Storybook Hub_

**Comment by richarddubay** [comment]
@jbaxleyiii @delianides I think this is a better solution. I just started over with a new branch.

File: imports/components/@primitives/layout/global/index.js [hunk 1]
```diff
@@ -157,34 +157,61 @@ export const URL_TITLE_QUERY = gql`
...
   }
 
   componentWillUnMount() {
     if (Meteor.isCordova) universalLinks.unsubscribe("universalLinkRoute");
   }
 
+  deviceReadyFunction = (saveDeviceRegistrationId) => {
+    universalLinks.subscribe("universalLinkRoute", this.universalLinkRouting);
+    /* eslint-disable */
+    if(isIOS()) {
```
Comment (jbaxleyiii) [inline imports/components/@primitives/layout/global/index.js]: @richarddubay lets one line this and fix the spacing between if and (

File: imports/components/@primitives/layout/global/index.js [hunk 2]
```diff
@@ -157,34 +157,61 @@ export const URL_TITLE_QUERY = gql`
...
     }
   }
 
   componentWillUnMount() {
     if (Meteor.isCordova) universalLinks.unsubscribe("universalLinkRoute");
   }
 
+  deviceReadyFunction = (saveDeviceRegistrationId) => {
+    universalLinks.subscribe("universalLinkRoute", this.universalLinkRouting);
+    /* eslint-disable */
```
Comment (jbaxleyiii) [inline imports/components/@primitives/layout/global/index.js]: in fact, why do we need to disable? We can add FCMPlugin to the eslint globals section

**Pull request merged**